index.js

```diff
@@ -0,0 +1,49 @@
+const pino = require("pino");
+
+/**
+ * List of default field that should not be transformed by the serializer
+*/
+const whitelist = ["time", "pid"];
+
+/**
+ * Replace the standard serializer to be able to transform each log
+ * Pino Documentation: https://getpino.io/#/docs/api?id=serializerssymbolfor39pino39-function
+*/
+const serializers = {};
+serializers[Symbol.for('pino.*')] = ( obj ) => {
+    Object.keys(obj).forEach(key => {
+        if (whitelist.indexOf(key) === -1) {
+            // Try to stringify objects/array or transform other types in string
+            try {
+                if (typeof obj[key] === "object" || obj[key] instanceof Array) {
+                    obj[key] = JSON.stringify(obj[key]);
+                } else {
+                    obj[key] = String(obj[key]);
+                }
+            } 
+            // in case of errors just return the current value
+            catch(err) {
+                obj[key] = obj[key];
+            }
+        }
+    });
+    return obj;
+};
+
+
+/**
+* Factory to create the logger
+*
+* @param {string} type    Which type of logger you want to create: "app" or "access"
+*/
+const createLogger = (type = "app") => {
```

Comment:
It could be useful to support an initial set of properties `props` instead of the `type` parameter. Something like:

```
createLogger({ type: `adserver`, context: `app` })
```

this is because: 

- `type` is required by our logger policy to identify the application itself
- `context` is required by our logger policy to dispatch that log to the appopriate destination

In this case, you can enable the serializers when `context !== 'access'`

Reply:
That make sense. I suggested myself to keep the first PR minimal.

Since we are talking about the factory params, the question is:

- do we want to enforce parameters validation at runtime? I think that a fail fast approach is better in general, but we are talking about logging here.

Reply:
Talking about this probably we should also handle our `default configuration` for `Pino` in the Factory.
```
const createLogger = (props, level) => {
    // Add the custom serializer if we are creating an AppLogger
    const options = {
        messageKey: "message",
        base: null,
        timestamp: true,
        level: level || "info",
    }
    if (props.context === "app") {
        options.serializers = serializers;
    }
    return pino(options).child(props);
};
```
What do you think @roccozanni?

Reply:
Absolutely, not sure how I could have missed in the review 😅